tests/test_2D.py

```diff
@@ -0,0 +1,21 @@
+import subprocess
+import sys
+from pathlib import Path
+
+from .util import get_data
+
+
+def test_output_2D_file_format() -> None:
+    """A simple test for format tag fix for 2D output arrays."""
+
+    Path("filename.txt").touch()
+    params = [
+        sys.executable,
+        "-m",
+        "cwltool",
+        "--cachedir",  # just so that the relative path of file works out
+        "foo",
```

Comment:
```suggestion
        "--cachedir",  # just so that the relative path of file works out
        str(tmp_path),
```
combined with using the `tmp_path` Pytest fixture as previously mentioned: https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html

Reply:
@mr-c  I think, I understood and followed the fixture pattern. I am not sure it is exactly what is needed. I hope the test passes :)